### PR TITLE
RELATED: RAIL-2107 - Merges from v7

### PR DESCRIPTION
**Pull request by lupko**
Merges of code from july:

- pivot table resizing fixes
- syntax highlighting input fix

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](docs/contributing.md#what-should-the-commits-look-like)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/184/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/1776/ : SUCCESS in 12m 05s

File: libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/pivotTable/tests/sortMocks.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
+// (C) 2020 GoodData Corporation
+
+// sortItems based on referencePointMocks.simpleStackedReferencePoint
+import { IAttributeSortItem, IMeasureSortItem } from "@gooddata/sdk-model";
+
+export const validMeasureSort: IMeasureSortItem = {
```
Comment (no23reason) [inline libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/pivotTable/tests/sortMocks.ts:6]: Should we use the factories here?

File: libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/pivotTable/tests/sortMocks.ts [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    measureSortItem: {
```
Comment (no23reason) [inline libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/pivotTable/tests/sortMocks.ts:7]: Should we use factories here?

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/198/ : SUCCESS in 18s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/453/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/1780/ : SUCCESS in 15m 09s

**Pull request merged**